src/style.css

```diff
@@ -1,3 +1,87 @@
 body {
   margin: 0;
+  padding: 0;
+  --color-blue: #00acdc;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  align-content: center;
```

Comment:
This would not work, because by default `flex-wrap: nowrap`. Redundant.

Reply:
removed align-content